src/de.rs

```diff
@@ -311,6 +311,12 @@ impl<'de> de::Deserializer<'de> for Deserializer {
             visitor.visit_string(v)
         } else if Array::is_array(&self.value) {
             self.deserialize_seq(visitor)
+        } else if self.value.is_instance_of::<Uint8Array>()
```

Comment:
Nit: let's reuse `as_bytes` here instead - it both avoids duplicating the type checks and calls into the more narrow `visit_byte_buf`.

Reply:
++